checks.d/zk.py

```diff
@@ -210,4 +317,34 @@ def parse_stat(self, buf):
         _, value = buf.readline().split(':')
         metrics.append(('zookeeper.nodes', long(value.strip())))
 
-        return metrics, tags, mode
+        return metrics, tags, mode, version
+
+
+    def parse_mntr(self, buf):
+        ''' `buf` is a readable file-like object
+            returns a tuple: (metrics, mode)
+            if mode == 'inactive', metrics will be None
+        '''
```

Comment:
If your docstring is going to follow the Python spec, `return` should be wrapped in colons. `:return:`

https://pythonhosted.org/an_example_pypi_project/sphinx.html#function-definitions